spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerAutoConfiguration.java

```diff
@@ -35,13 +35,15 @@
 @AutoConfigureOrder(Ordered.HIGHEST_PRECEDENCE)
 public class LocalDeployerAutoConfiguration {
 
-	@Bean
+	public static final String DEPLOYER = "Local";
```

Comment:
Bean names typically begin with a lowercase letter. Is there a reason you want this to start with uppercase?